src/api_proxy/service_control/request_info.h

```diff
@@ -167,7 +176,7 @@ struct CheckResponseInfo {
       : consumer_project_number(),
         consumer_type(),
         consumer_number(),
-        error({"", false, ScResponseErrorType::ERROR_TYPE_UNSPECIFIED}),
+        error(),
```

Comment:
do we need to list it out if a member variable is using default constructor? 

can we remove this constructor and use initializer "=" for api_key_state